src/app/app-routing.module.ts

```diff
@@ -1,6 +1,6 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { CollectionPage, FormsPage, PhnHeaderPage, UtilitiesPage } from './pages';
+import * as fromPages from './pages';
 
 
 const routes: Routes = [
```

Comment:
Looking at this again, there is some redundancy with the `Route` type defined `app.component.ts`.
We could probably extend the angular `Route` like `type ExtendedRoute = Route & { name?: string };` and then define all the names here and export routes in order to reuse it in `app.component.ts`.